tests/selfapps_layered_e2e.ps1

```
@@ -0,0 +1,364 @@
+            $exeTimeoutMs = 60000
+            if ($proc.WaitForExit($exeTimeoutMs)) {
+                $exeExit = $proc.ExitCode
+            } else {
+                # derived requirement: Process.Kill() (the parameterless overload) terminates
+                # ONLY $proc itself -- a onefile bootloader (or any program) that spawns a child
+                # inheriting the redirected stdout/stderr handles can leave that child running
+                # after $proc is killed, so the pipe never reaches EOF and an unbounded
+                # ReadToEndAsync().Result would hang forever, defeating the whole point of this
+                # bounded launch. Same taskkill /T (process-tree kill) + bounded-drain pattern
+                # already established in tools/exe_hint_rerun.ps1 for the identical hazard.
+                try { & taskkill.exe /F /T /PID $proc.Id 2>$null 1>$null } catch {}
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)